server/src/test/java/org/elasticsearch/common/io/stream/AbstractStreamTests.java

```diff
@@ -222,7 +222,11 @@ public void testSmallBigIntArray() throws IOException {
     }
 
     public void testLargeBigIntArray() throws IOException {
-        assertBigIntArray(between(PageCacheRecycler.INT_PAGE_SIZE, 10000));
+        assertBigIntArray(between(PageCacheRecycler.INT_PAGE_SIZE, 5_000_000));
```

Comment:
I originally thought a bigger value would yield it. It didn't, but the bigger value doesn't take too long anyway.